tests/appsec/iast/test_iast.py

```diff
@@ -12,7 +12,7 @@
     java={"spring-boot": "0.108.0", "spring-boot-jetty": "0.108.0", "*": "?"},
     nodejs={"express4": "3.6.0", "*": "?"},
     php_appsec="?",
-    python="?",
+    python="1.6.0rc1.dev",
```

Comment:
This version does not exists yet. It means that the test is not executed. And once it'll exists, if the tracers does not pass the test, your CI will be broken (among a lot of other CI).

FYI, current dev version from your `main` branch is python@1.5.0rc2.dev105+g61f01775 : https://github.com/DataDog/system-tests/actions/runs/3386289779/jobs/5625523608#step:9:31